### ISSUE TITLE
`gts check/fix` uses a list of files created by tslint

### ISSUE THREAD

**Issue by kjin** [google/ts-style#98]
See below comment

~On my mac, `gts check` exits with code 0 before checking all files [here](https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/621). On linux this doesn't happen, and I get my clang-format error reports.~

~I combed `gts` dependencies for `process.exit` calls but couldn't find any. I believe unresolved Promises are somehow being ignored, and the event loops drains fully. Unsure about this...~

**Comment by ofrobots** [comment]
I'm unable to reproduce this on your `plugin-types` branch. You will have to do some initial investigation yourself and report back.

**Comment by kjin** [comment]
The issue stems from me including generated git-ignored source files, which are being checked. I didn't realize that I had already fixed them locally, and CI kept failing because it was regenerating these source files.

This makes me think of two improvements we can make to avoid this problem (despite the questionable usage of generated source files in the linked PR, in general it's a use case we still want to support):
* `gts check` and `gts fix` should show a list of files that `clang-format` takes issue with, if not the fixes that should be made/have been made themselves. I already talked about this in #60 as being a low-priority improvement but I now think this would be extremely useful.
* We should not use the list of files generated by `tslint` for formatting (this is the current behavior). It seems like `Linter.getFileList(Linter.createProgram('...'))` gets a list of files in a source dependency graph, making its return value a longer list than the list of files matching the glob specified in `tsconfig.json`. This is useful for linting, but undesirable for formatting.
